svm/tests/example-programs/write-to-account/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+use solana_program::{
+    account_info::{next_account_info, AccountInfo},
+    entrypoint,
+    entrypoint::ProgramResult,
+    incinerator, msg,
+    program_error::ProgramError,
+    pubkey::Pubkey,
+};
+
+entrypoint!(process_instruction);
+
+fn process_instruction(
+    _program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    data: &[u8],
+) -> ProgramResult {
+    let accounts_iter = &mut accounts.iter();
+    let target_account_info = next_account_info(accounts_iter)?;
+    match data[0] {
+        // print account size
+        0 => {
+            msg!(
```

Comment:
Do you print here for debug purpose or to imitate reading the data without writing back? In case of the second, I believe `msg` cost is not negligible and instead I would call `std::ptr::read_volatile` which prevents compiler to optimize unused code (which doesn't happen with our compiler anyways).